pkg/rpm_pfg.bzl

```diff
@@ -352,10 +352,14 @@ def _process_subrpm(ctx, rpm_name, rpm_info, rpm_ctx):
             "%%files %s" % rpm_info.package_name,
         ]
 
-        for dep in rpm_info.srcs:
-            _process_dep(dep, sub_rpm_ctx)
+        if rpm_info.srcs:
+            for dep in rpm_info.srcs:
+                _process_dep(dep, sub_rpm_ctx)
+
+            rpm_lines += sub_rpm_ctx.rpm_files_list
+        else:
+            rpm_lines += "%defattr(-,root,root)"
```

Comment:
Should you just write this as the first line in all cases? Would that be wrong?

Reply:
That is one possibility.  Do we want to parameterize this though so that we can specify some useful defaults or just leave it as a contant?

Reply:
Oh, now you are asking hard questions. I think a constant is fine because IIRC we write the mode for every file.
But, now I am less certain.
How about just adding a comment about why we emit the defattr and that perhaps we could add it. And some future person will make that choice when they want to cleanup that code.

Reply:
Done